modules/analysis-common/src/test/java/org/elasticsearch/analysis/common/CommonAnalysisFactoryTests.java

```diff
@@ -56,6 +56,15 @@ public CommonAnalysisFactoryTests() {
         filters.put("trim",                  TrimTokenFilterFactory.class);
         filters.put("worddelimiter",         WordDelimiterTokenFilterFactory.class);
         filters.put("worddelimitergraph",    WordDelimiterGraphTokenFilterFactory.class);
+        filters.put("flattengraph",          FlattenGraphTokenFilterFactory.class);
```

Comment:
I believe @andy-elastic started a trend to remove the extra spaces. Can you remove the extra spaces in this map while you are modifying it?